src/core/src/core_logic/PatchInstaller.py

```diff
@@ -800,4 +810,20 @@ def get_max_batch_size(self, maintenance_window, package_manager):
         self.composite_logger.log_debug("Calculated max batch size is: {0}".format(max_batch_size_for_packages))
 
         return max_batch_size_for_packages
-
+    
+    def is_install_patches_in_single_batch_enabled(self):
+        """"""
+        if self.install_patches_in_single_batch == True:
+            return self.install_patches_in_single_batch
+        
+        dur = datetime.datetime.strptime(self.execution_config.duration, "%H:%M:%S")
+        dura = timedelta(hours=dur.hour, minutes=dur.minute, seconds=dur.second)
+        total_time_in_minutes = self.env_layer.datetime.total_minutes_from_time_delta(dura)
+        max_maintenance_window_criteria = total_time_in_minutes >= Constants.MAX_PATCH_OPERATION_DURATION_IN_MINUTES
```

Comment:
Add a line break after these 4 lines, and a comment on top of the "para" for one-line readability of what the section is doing - 'evaluating Maintenance Window criteria;